registry/app.go

```diff
@@ -88,6 +88,8 @@ func NewApp(configuration configuration.Configuration) *App {
 }
 
 func (app *App) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	// Set a header with the Docker Distribution API Version for all responses.
+	w.Header().Set("Docker-Distribution-API-Version", "2.0")
```

Comment:
You're absolutely right, my mind must have drifted off, had my mind on people testing builds from GitHub during development .